content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -11,104 +11,241 @@ keywords:
 - data backup and recovery
 ---
 
-Google Photos works best in TrueNAS using a Google Photos API key and [rclone](https://rclone.org/) token.
+Google Photos cloud sync tasks in TrueNAS SCALE use the [rclone](https://rclone.org/) backend for the [Google Photos API](https://developers.google.com/photos) to authenticate credentials and transfer data.
+
+Configuring a Google Photos cloud sync task is a multi-part procedure that includes:
+
+1. [Planning your deployment](#before-you-begin) and selecting a local dataset.
+2. [Generating Google API credentials](#creating-the-api-credentials) on the Google Cloud API dashboard.
+3. [Installing rclone and generating a token](#configuring-rclone) on your remote client OS.
+4. [Adding Google Photos cloud credentials](#adding-google-photos-cloud-credentials) on TrueNAS SCALE.
+5. [Configuring the cloud sync task](#creating-the-cloud-sync-task) on SCALE.
+
+## Before You Begin
+
+Review your storage and data protection requirements and consider your options before setting up a Google Photos cloud sync task.
+Refer to the rclone Google Photos backend documentation for more information on using rclone to sync Google Photos, including [standard options](https://rclone.org/googlephotos/#standard-options) and [limitations of the Google Photos API](https://rclone.org/googlephotos/#limitations), that might help you plan your deployment.
+
+Consider how you want to manage your media files on Google Photos and in your local dataset.
+Select the cloud sync task [direction and transfer mode](#choosing-a-sync-direction-and-mode), [remote folder](#choosing-a-target-folder) to target, and [new or existing local dataset](#selecting-the-dataset-and-organizing-files) to pull to or push from, that best fit your needs.
```

Comment:
Because this section is talking about planning, it makes more sense to say:
```suggestion
Decide on the cloud sync task [direction and transfer mode](#choosing-a-sync-direction-and-mode), [remote folder](#choosing-a-target-folder) to target, and [new or existing local dataset](#selecting-the-dataset-and-organizing-files) to pull to or push from, that best fit your needs.
```
Thoughts?